app/controllers/Ublog.scala

```diff
@@ -130,11 +130,19 @@ final class Ublog(env: Env) extends LilaController(env):
 
   def update(id: UblogPostId) = AuthBody { ctx ?=> me ?=>
     NotForKids:
-      Found(env.ublog.api.findEditableByMe(id)): prev =>
+      Found(
+        env.ublog.api
+          .findEditableByMe(id)
+          .flatMapz: prev =>
+            env.user.repo
+              .byId(prev.created.by)
+              .map: userOpt =>
+                userOpt.map(user => (user, prev))
+      ): (author, prev) =>
```

Comment:
the scala could mos probably be improved here, struggled a bit and quite tired today